receiver/prometheusreceiver/testdata/config_k8s.yaml

```diff
@@ -0,0 +1,40 @@
+receivers:
```

Comment:
I'd be happy to share a complete DaemonSet deployment example with Prometheus receiver based on our usage at Google, if there is interest.

There is https://github.com/open-telemetry/opentelemetry-collector/blob/master/examples/k8s.yaml but it's not using Prometheus.